docs/output.md

```diff
@@ -78,7 +107,9 @@ The pipeline has special steps which allow the software versions used to be repo
 
 * `multiqc/Project_multiqc_report.html`  
   MultiQC report - a standalone HTML file that can be viewed in your web browser
-* `multiqc/Project_multiqc_data/`  
+* `multiqc/multiqc_data/`  
   Directory containing parsed statistics from the different tools used in the pipeline
+* `multiqc/multiqc_plots/`
+  Directory containing the image files of the graphs included in MultiQC
```

Comment:
We should also include a description of the `pipeline_info` files generated by Nextflow. See https://github.com/nf-core/chipseq/blob/master/docs/output.md#other-results